### DISCUSSION TITLE
Xiaomi Mi Router 3 Pro (архитектура не поддерживается)

### DISCUSSION THREAD

**Discussion by EvgeniyNN** [bol-van/zapret#768] [General]
ramips/mt7621
Linux OpenWrt 5.15.167 #0 SMP Mon Sep 23 12:34:46 2024 mips GNU/Linux
# opkg print-architecture
arch all 1
arch noarch 1
arch mipsel_24kc 10

Run ./install_bin.sh
Error:
no compatible binaries found
помогите установить на этот роутер.

**Comment by bol-van** [comment]
Релиз скачивайте. Бинарики из репы убраны

**Reply by EvgeniyNN** [reply]
Весьма благодарен!
сейчас попробую.
ps вы бы хоть поправили [инструкцию](https://github.com/bol-van/zapret/blob/master/docs/quick_start.md) тогда

**Reply by bol-van** [reply]
Там поправлено.

> Скачайте последний [tar.gz релиз](https://github.com/bol-van/zapret/releases) в /tmp, распакуйте его, затем удалите архив.

**Reply by M-A-M-B-A** [reply]
Прошу добавить команды для скачивания и распаковки

ro```
ot@OpenWrt:/tmp# wget -4 -c https://github.com/bol-van/zapret/releases/download/v68/zapret-v68.tar.gz
Downloading 'https://github.com/bol-van/zapret/releases/download/v68/zapret-v68.tar.gz'
Connecting to 140.82.121.3:443
Redirected to /github-production-release-asset-2e65be/51757458/3fe27af5-70bd-4a4d-9008-dd92c95b237e?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20241112%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20241112T133405Z&X-Amz-Expires=300&X-Amz-Signature=1f46b28f7d131f301244e9c6964453605be0730a035c2e94834c99e755fc64fd&X-Amz-SignedHeaders=host&response-content-disposition=attachment%3B%20filename%3Dzapret-v68.tar.gz&response-content-type=application%2Foctet-stream on objects.githubusercontent.com
Writing to '3fe27af5-70bd-4a4d-9008-dd92c95b237e?X-Amz-Algorithm=AWS4-HMAC-SHA256'
3fe27af5-70bd-4a4d-9 100% |*******************************|  5749k  0:00:00 ETA
Download completed (5887697 bytes)

```

**Reply by bol-van** [reply]
Если человек полез на openwrt, он должен знать как скопировать файл.
Иначе много нытья и нянек обеспечено. Там прям в начале написано. Не умеешь - не лезь

**Reply by M-A-M-B-A** [reply]
`curl -LJO https://github.com/bol-van/zapret/releases/download/v68/zapret-v68.tar.gz`
